cvat-ui/src/components/annotation-page/annotations-actions/annotations-actions-modal.tsx

```diff
@@ -414,6 +438,7 @@ function AnnotationsActionsModalContent(props: { onClose: () => void; }): JSX.El
                                         value={frameFrom}
                                         min={(jobInstance as Job).startFrame}
                                         max={frameTo}
+                                        disabled={currentFrameAction}
```

Comment:
There is no reason to render this section at all if only current frame allowed

Reply:
Maybe to render simple text like: "Running the action is only allowed on current frame"

Reply:
Added simplier text